app-android-xamarin/Moodis/Moodis/Feature/Group/GroupActivityModel.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Moodis.Constants.Enums;
+using Moodis.Database;
+using Moodis.Feature.Register;
+using Moodis.Feature.SignIn;
+
+namespace Moodis.Feature.Group
+{
+    class GroupActivityModel
+    {
+        public static List<Group> groups = DatabaseModel.FetchGroupFromDatabase();
+
+        public Response AddUserToGroup(string groupName)
+        {
+            if(groups != null)
+            {
+                var username = SignInViewModel.currentUser.Username;
+                var group = groups.Find(groupTemp => groupTemp.Groupname == groupName);
+                if(group == null)
+                {
+                    return Response.UserNotFound;
+                }
+                if(!group.IsMember(username))
+                {
+                    group.AddMember(username);
+                    return Response.OK;
+                }
+                else
+                {
+                    return Response.UserExists;
+                }
+            }
+            else
+            {
+                return Response.GeneralError;
+            }
+        }
+
+        public Response CreateGroup(string groupName)
+        {
+            if(!groups.Exists(group => group.Groupname == groupName))
+            {
+                var newGroup = new Group(groupName, SignInViewModel.currentUser.Username);
+                groups.Add(newGroup);
+                DatabaseModel.AddGroupToDatabase(newGroup);
+                return Response.OK;
+            }
+            else
+            {
+                return Response.UserExists;
+            }
+        }
+
+        public static List<string> GetFriendIds()
```

Comment:
why use static ?

Reply:
Because it is used in other classes and I did not want to add another Model that is not meant directly for that class